Bindings/Java/Matlab/configureOpenSim.m.in

```diff
@@ -114,6 +114,7 @@ try
         % This function returns all but the leaf element of the provided path.
         openSimFolder = fileparts(openSimFolder);
     end
+    api_distro = false;
```

Comment:
Minor, but it would be clearer to initialize `api_distro` in `uiGetOpenSimFolder` and return it as an additional output.